configuration/ldap/extra.py

```diff
@@ -1,8 +1,28 @@
 ####
 ## This file contains extra configuration options that can't be configured
 ## directly through environment variables.
+## All vairables set here overwrite any existing found in ldap_config.py
 ####
 
-# import ldap
+# # This Python script inherits all the imports from ldap_config.py
+# from django_auth_ldap.config import LDAPGroupQuery # Imported since not in ldap_config.py
 
-# AUTH_LDAP_MIRROR_GROUPS = ["netbox-group-1", "netbox-group-2"]
+# # Sets a base requirement of membetship to netbox-user-ro, netbox-user-rw, or netbox-user-admin.
+# AUTH_LDAP_REQUIRE_GROUP = (
+#     LDAPGroupQuery("cn=netbox-user-ro,ou=groups,dc=example,dc=com")
+#     | LDAPGroupQuery("cn=netbox-user-rw,ou=groups,dc=example,dc=com")
+#     | LDAPGroupQuery("cn=netbox-user-admin,ou=groups,dc=example,dc=com")
+# )
+
+# # Sets LDAP Flag groups variables with example.
+# AUTH_LDAP_USER_FLAGS_BY_GROUP = {
+#     "is_staff": (
+#         LDAPGroupQuery("cn=netbox-user-ro,ou=groups,dc=example,dc=com")
+#         | LDAPGroupQuery("cn=netbox-user-rw,ou=groups,dc=example,dc=com")
+#         | LDAPGroupQuery("cn=netbox-user-admin,ou=groups,dc=example,dc=com")
+#     ),
+#     "is_superuser": "cn=netbox-user-admin,ou=groups,dc=example,dc=com",
+# }
+
+# # Sets LDAP Mirror groups variables with example groups
+# AUTH_LDAP_MIRROR_groups = ["netbox-user-ro", "netbox-user-rw", "netbox-user-admin"]
```

Comment:
Shouldn't "AUTH_LDAP_MIRROR_groups" be "AUTH_LDAP_MIRROR_GROUPS"?

Reply:
@ryanmerolle can you comment on this?

Reply:
Good catch, how should I resolve since this was merged into the PR.  It still works either way if I am not mistaken as I pulled it from a running example.

Reply:
You can just create a new PR targeting the `develop` branch.

Reply:
Once we merge your PR into `develop`, it will automatically become part of this release.

Reply:
#499 corrects this